docusaurus/docs/reference/ha/bootstrapping.md

```diff
@@ -0,0 +1,164 @@
+
+```yaml
+ctrl:
+  listener: tls:0.0.0.0:6262
+  options:
+    advertiseAddress: tls:192.168.1.100:6262
+```
+ 
+Finally, for sessions to work across controllers, JWTs are used. To enable these
+an OIDC endpoint should be configured.
+
+```yaml
+web:
+  - name: all-apis-localhost
+    bindPoints:
+      - interface: 127.0.0.1:1280
+        address: 127.0.0.1:1280
+    options:
+      minTLSVersion: TLS1.2
+      maxTLSVersion: TLS1.3
+    apis:
+      - binding: health-checks
+      - binding: fabric
+      - binding: edge-management
+      - binding: edge-client
+      - binding: edge-oidc
+```
+
+## Initializing the Controller
+
+Once properly configured, the controller can be started.
+
+```shell
+ziti controller run ctrl1.yml
+```
+
+Once the controller is up and running, it will see that it is not yet initialized, and will pause
+startup, waiting for initialization. While waiting it will periodically emit a message:
+
+```
+[   3.323] WARNING ziti/controller/server.(*Controller).checkEdgeInitialized: the 
+Ziti Edge has not been initialized, no default admin exists.  Add this node to a 
+cluster using 'ziti agent cluster add tls:localhost:6262' against an existing 
+cluster member, or if this is the bootstrap node, run 'ziti agent controller init' 
+to configure the default admin and bootstrap the cluster
+```
+
+As this is the first node in the cluster, we can't add any nodes to it yet. Instead, run:
+
+```
+ziti agent controller init <admin username> <admin password> <admin identity name>
+```
+
+This initializes an admin user that can be used to manage the network.
+ 
+## Managing the Cluster
+
+There are four commands which can be used to manage the cluster.
+
+```bash
+# Adding Members
+ziti agent cluster add <other controller raft address>
+
+# Listing Members
+ziti agent cluster list
+
+# Removing Members
+ziti agent cluster remove <controller id>
+
+# Transfer Leadership
+ziti agent cluster transfer-leadership [new leader id]
+```
+
+These are also available via the REST API, and can be invoked through the CLI.
+
+```bash
+$ ziti ops cluster --help
+Controller cluster operations
+
+Usage:
+  ziti ops cluster [flags]
+  ziti ops cluster [command]
+
+Available Commands:
+  add-member          add cluster member
+  list-members        list cluster members and their status
+  remove-member       remove cluster member
+  transfer-leadership transfer cluster leadership to another member
+
+Flags:
+  -h, --help   help for cluster
+
+Use "ziti ops cluster [command] --help" for more information about a command.
+```
+
+## Growing the Cluster
+
+Once a single node is up and running, additional nodes can be added to it. They should be 
+configured the same as the initial node, though they will have different addresses.
+
+The first node, as configured above, is running at `192.168.1.100:6262`. 
+
+If the second node is running at `192.168.1.101:6262`, then it can be added to the 
+cluster in one of two ways. 
+
+### From An Existing Node
```

Comment:
IMO the titles break up the flow here too much. Persaonlly, I'd rather see two bullets instead but -- whatever :) it's a nit...

Also "we" shows up here twice and should be reworked